src/process/errors.rs

```diff
@@ -1,24 +1,22 @@
 use std::io;
 
-use snafu::Snafu;
-
 use crate::{Error, Pid};
 
 pub type ProcessResult<T> = std::result::Result<T, ProcessError>;
 
-#[derive(Debug, Snafu)]
-#[snafu(visibility(pub(crate)))]
+// TODO: get this visibility junk sorted out
```

Comment:
Is this still an issue? I didn't think thiserror had to deal with any visibility shenanigans.

Reply:
Ah yeah, sorry I forgot to clean up that comment before I submitted. I can remove tomorrow.

Reply:
Just pushed a commit to fix this.